db/patterns/inmarsat.eno

```diff
@@ -6,6 +6,7 @@ organization: viasat
 --- domains
 inmarsat.com
 wifi.koreanair.com
+access-wifi.net
```

Comment:
This is a bit tricky to me how to deal with it. As I understand, it is only available if you are already connected to the network?

For "wifi.koreanair.com", it is likely the same, but wifi.koreanair.com has the advantage that the name provides some protection (subdomain of koreanair.com). For "access-wifi.net", I wonder if it could clash with names from other (private) networks.

Would be easier if we had a source to back it up. I tried to find something about the ownership of the domain, but unsuccessfully. Only that it is registered via Amazon.

Reply:
I also don't think it has a public record that saying itself is owned by any organisation unfortunately. When I try to decide about those kinds of private networks, I try to access "ip.me" to check the ISP.

This one would be complicated and open question for now because `wifi-access.net` is only involved for payment. I didn't make a screenshot for that.